icons/mastodon.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="61.076954mm" height="65.47831mm" viewBox="0 0 216.4144 232.00976">
```

Comment:
Could you make this icon black & white and the same size as the others? see fb for an example: https://github.com/dschep/proven/blob/master/icons/facebook.svg